dot/core/service_integration_test.go

```diff
@@ -128,11 +60,16 @@ func TestAnnounceBlock(t *testing.T) {
 	err = digest.Add(*prd)
 	require.NoError(t, err)
 
+	// Used to define the state root of new block for testing
+	genesisBlock, err := s.blockState.GetBlockByHash(s.blockState.GenesisHash())
```

Comment:
nit Can we call something more specific to set the state root of our test block? Or not, just a quick question

Reply:
I dont think so. Looked around and didn't see anything that jumped out at me